src/pages/[platform]/build-a-backend/data/working-with-files/index.mdx

```diff
@@ -1078,15 +1078,16 @@ import PhotosUI
 struct WorkingWithFilesApp: App {
 
     init() {
+        _ = AWSAPIPlugin(modelRegistration: AmplifyModels())
         do {
             Amplify.Logging.logLevel = .verbose
             try Amplify.add(plugin: AWSCognitoAuthPlugin())
+            try Amplify.add(plugin: AWSAPIPlugin())
```

Comment:
remove first AWSAPIPlugin and update second one to `try Amplify.add(plugin: AWSAPIPlugin(modelRegistration: AmplifyModels()))`